Sources/SwiftDocC/SwiftDocC.docc/Resources/ThemeSettings.spec.json

```
@@ -856,6 +856,9 @@
                                     },
                                     "onThisPageNavigator": {
                                         "$ref": "#/components/schemas/FeatureFlagDisable"
+                                    },
+                                    "i18n": {
+                                        "$ref": "#/components/schemas/FeatureFlagDisable"
```

i18 is defined by "enable"
```suggestion
                                        "$ref": "#/components/schemas/FeatureFlagEnable"
```